src/components/PostPreview/PostPreview.tsx

```diff
@@ -0,0 +1,77 @@
+import type Post from '@types/Post';
+import { Avatar } from '../Avatar';
+import Icon from '../Icon';
+import {
+  PreviewWrapper,
+  PostHeaderWrapper,
+  PostInfoWrapper,
+  PostContent,
+  UserNameWrapper,
+  AvatarWrapper,
+  PostDetailInfoWrapper
+} from './PostPreview.style';
+
+interface PostPreviewProps {
+  post: Post;
+  noneProfile: boolean;
+}
```

Comment:
```suggestion
interface PostPreviewProps {
  previewPostData: Pick<
    Post,
    'image' | 'likes' | 'comments' | 'title' | 'author' | 'createdAt'
  >;
  noneProfile: boolean;
}
```
이런식으로 컴포넌트 내부에서 실제로 사용하는 데이터만 prop으로 받아야하지 않을까요? 
지금은 PostPreview컴포넌트를 사용하려면 팔로워나 팔로잉같은 데이터도 넘겨줘야 사용할 수 있는 상태인 것 같아요!

Reply:
아하..! Pick 을 사용해서 받을 수 있군요! 배워갑니다!